fern/pages/servers/express.mdx

```diff
@@ -0,0 +1,143 @@
+
+## What Fern generates
+
+- TypeScript interfaces for your API types
+- Exceptions that you can throw for non-200 responses
+- Interfaces for you to define your business logic
+- All the networking/HTTP logic to call your API
+
+## Use the starter repository 
+
+Clone the [starter repo](https://github.com/fern-api/express-starter/) for Express + React. 
+
+## Demo video
+
+Learn how Fern can be helpful when building an Express server. 
+
+<div style="position: relative; padding-bottom: 62.5%; height: 0;"><iframe src="https://www.loom.com/embed/31f4243c4d824c54938bdc4840fbb8ba?sid=3f460e48-af22-4db3-be2f-48884ab0170d" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen style="position: absolute; top: 0; left: 0; width: 100%; height: 100%;"></iframe></div>
+
+## Getting started 
+
+<Steps>
+
+### Define your API 
+
+Define your API using the [OpenAPI Specification](/learn/api-definition/openapi/overview) or the [Fern Definition](/learn/api-definition/fern/overview).
+
+### Add the Express generator 
+
+Add the Express server generator to your `generators.yml` file.
+
+<CodeBlock title="generators.yml">
+```yaml
+groups:
+  express-server:
+    generators:
+      - name: fernapi/fern-typescript-express
+        version: 0.15.0
+        output:
+            location: local-file-system
+            path: ../app/server
+```
+</CodeBlock>
+
+### 
+Run the [CLI](/learn/cli-api/cli-reference/cli-overview) command `fern generate --group express-server`. 
+
+<Note>Make sure to enable `allowSyntheticDefaultImports` in your `tsconfig.json` when using this generator.</Note>
+
+### Implement the functions 
+
+Provide the business logic by implementing each function. For example, if we were implenting the IMDb API, we would implement the `get_movie` function.
+
+<CodeBlock title="src/movies_service.py">
+```python
+from .generated.fern import AbstractMoviesService, Movie, MovieDoesNotExistError, MovieId
+
+class MoviesService(AbstractMoviesService):
+    def get_movie(self, *, movie_id: str) -> Movie:
+        if movie_id == "titanic":
+            return Movie(
+                id=MovieId.from_str("titantic"),
+                title="Titanic",
+                rating=9.8,
+            )
+        raise MovieDoesNotExistError(MovieId.from_str(movie_id))
+```
+</CodeBlock>
+
+### Run the server
+
+```bash
+poetry run start
+```
+
+### Call the API 🚀
+
+```bash
+$ curl --location --request GET --silent 'localhost:8080/movies/titanic' | jq .
+{
+  "id": "titantic",
+  "title": "Titanic",
+  "rating": 9.8
+}
+
+$ curl --location --request GET --silent 'localhost:8080/movies/oceans-11' | jq .
+{
+  "error": "MovieDoesNotExistError",
+  "errorInstanceId": "f6e1d69c-bf97-42d5-bc89-5e42773e3880",
+  "content": "oceans-11"
+}
+```
+
+</Steps>
+
+## Advanced Configurations
```

Comment:
@zachkirsch good ol' server stubs coming back into play

Reply:
lol love it. I think I have “server INTERFACES not stubs” still burned into my memory